SpatialGDK/Source/Private/Interop/SpatialSender.cpp

```diff
@@ -279,12 +286,24 @@ void USpatialSender::SendComponentInterest(AActor* Actor, Worker_EntityId Entity
 
 void USpatialSender::SendPositionUpdate(Worker_EntityId EntityId, const FVector& Location)
 {
+	if (!NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::POSITION_COMPONENT_ID))
```

Comment:
Is this overkill? We shouldn't be replicating the actor if we don't have authority.